src/open_inwoner/configurations/bootstrap/utils.py

```diff
@@ -0,0 +1,82 @@
+from dataclasses import dataclass
+from typing import Any, Iterator
+
+from django.db.models.fields import NOT_PROVIDED
+
+
+@dataclass(frozen=True)
+class ConfigField:
+    name: str
+    verbose_name: str
+    description: str
+    field_type: str
+    default_value: str
+    values: str
+
+
+def populate_fields(
+    fields: dict[str, list],
+    require: tuple[str],
+    exclude: tuple[str],
+    all_fields: Iterator,
+):
+    for field in all_fields:
+        config_field = ConfigField(
+            name=field.name,
+            verbose_name=field.verbose_name,
+            description=field.description,
+            field_type=field.get_internal_type(),
+            default_value=get_default_value(field),
+            values=get_example_value(field),
+        )
+        fields["all"].append(config_field)
+        if require and config_field.name in require:
+            fields["required"].append(config_field)
+
+
+def get_default_value(field: Any) -> str:
+    default = field.default
+
+    if default is NOT_PROVIDED:
+        return "No default"
+    if not isinstance(default, (str, bool, int)):
+        return "No information"
```

Comment:
In which cases does this happen?

Reply:
Sometimes the default is a function. For example, for `DIGID_OIDC_OIDC_RP_SCOPES_LIST`, the default will be `&lt;function get_default_scopes_bsn at 0x7f0c994dcb80&gt;`

Reply:
I think in that case we could do a check using `callable` (https://docs.python.org/3/library/functions.html#callable) and in case it is callable we can call it to get the defaults

Reply:
Good idea, thanks!